spec/tic_tac_toe_wp_spec.rb

```diff
@@ -19,7 +19,11 @@
 
     expect(@tic_tac_toe_wp.is_there_a_winner?).to be true
   end
-
+end
+RSpec.describe TicTacToeWP::GameLogic do
```

Comment:
There should only be one describe block for the class, if you need to do different before each calls you can use `describe 'whatever you are describing' do` instead.

Reply:
Agreed, these were only added in to see if it would resolve the rubocop failure complaint. Reverting now.